app/views/shopping_lists/index.html.haml

```diff
@@ -0,0 +1,19 @@
+.d-grid.gap-2.d-md-block.mx-5
+  %button.btn.btn-dark.me-2.position-relative(type="button")= link_to 'New Shopping List', new_shopping_list_path, class: 'text-light text-decoration-none stretched-link'
+
+#shopping-lists-container.d-flex.my-2.mx-5.bg-light.position-relative
+  %nav#shopping-lists-index-navigation.d-flex.flex-column.align-items-start.p-2
+    - @shopping_lists.group_by{|li| li.status}.sort.reverse.each do |status, lists|
+      %h4.mt-2= status
+      - lists.each do |list|
+        .d-flex.align-items-center.list-group-item.list-group-item-action.p-1
+          .px-2.shc-icon{:class => list.status=='pending' ? "pending-icon" : "done-icon"}
```

Comment:
I think it should be possible to to use autogenerated `list.pending?` method for enum fields, please check if it will work here :)